assets/prototype/domain/shared/predicates/datetimes/capacityAtOrAbove.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { is } from 'ramda';
+
+/**
+ * Internal dependencies
+ */
+import { Datetime } from '../../../eventEditor/data/types';
+import validFiniteReglimit from './validFiniteReglimit';
+import validSold from './validStatus';
+
+/**
+ * @param {Object} date event date object
+ * @param {number} capacity
+ * @return {boolean} true if sold/regLimit >= capacity
+ */
+const capacityAtOrAbove = (date: Datetime, capacity) => {
+	return (
+		validSold(date) &&
+		validFiniteReglimit(date) &&
+		Math.round(date.sold) / Math.round(date.regLimit) >= capacity / 100
```

Comment:
In old version of `capacityAtOrAbove`, this was written this way:
```
parseInt( dateEntity.sold, 10 ) /
parseInt( dateEntity.regLimit, 10 ) >= ( capacity / 100 )
```

but TS is complaining about that: 
![image](https://user-images.githubusercontent.com/1477453/71365154-d4033400-25a6-11ea-8296-c0c14d35c88c.png)

and since we are enforcing via types that `sold` and `regLimit` should be numbers, I've used `Math.round`